tests/foreman/ui/test_http_proxy.py

```diff
@@ -17,11 +17,26 @@
 from fauxfactory import gen_integer
 from fauxfactory import gen_string
 from fauxfactory import gen_url
+from nailgun import entities
 
+from robottelo.constants import FAKE_0_PUPPET_REPO
+from robottelo.constants import FAKE_1_YUM_REPO
+from robottelo.constants import REPO_TYPE
+from robottelo.decorators import fixture
 from robottelo.decorators import tier2
 from robottelo.decorators import upgrade
 
 
+@fixture(scope='module')
+def module_org():
```

Comment:
This is already defined in robottelo/api/entity_fixtures.py and imported in conftest.py , use it from there instead of defining here.

Reply:
@jyejare If I use the approach you suggested then, I will have to add code for manually selecting the required org/loc when UI part of test starts executing otherwise the test will fail because it won't be able to find http proxy for that org/loc. But If I keep the current approach then I won't have to do any of that. Also most of the UI tests seems to be following the same approach.

Reply:
This is what I have observed so far, I will look further into it and if I find any other way then I will update the code accordingly. :)

Reply:
Probabbly one of your entity(may be http proxy) automatically creating different orgs/locs and hence the entity is not discoverable. You can take a look at this later if its taking more time to troubleshoot.

Running the tests ASAP that code quality is more important for now. You please revisit to fix this later.

Reply:
@jyejare yes probably, I will see why it's not working and fix it in another PR.